elfpy/data/acquire_data.py

```diff
@@ -171,10 +175,14 @@ def main(
                         time.sleep(1)
                         continue
 
-                if block_transactions is None:  # Proceed only if we have data, otherwise do nothing
+                if (
+                    block_transactions is None or wallet_deltas is None
+                ):  # Proceed only if we have data, otherwise do nothing
                     raise ValueError("Error in getting transactions")
```

Comment:
you didn't add this but can you remove it?  Totally expected that there may be some blocks without hyperdrive transactions, we shouldn't throw an error, just continue instead.